nomenclature/cli.py

```diff
@@ -154,3 +154,42 @@ def check_region_aggregation(
         results_df.to_excel(processed_data)
     if differences:
         differences_df.reset_index().to_excel(differences, index=False)
+
+
+@cli.command("export-project")
+@click.argument("path", type=click.Path(exists=True, path_type=Path))
+@click.argument("target", type=click.Path(path_type=Path))
+@click.option(
+    "--definitions",
+    help="Optional name for definitions folder",
+    type=str,
+    default="definitions",
+)
+@click.option(
```

Comment:
Same argument as above, I'd simplify the command by just fixing the exporting for all dimensions. Unless of course there's a good reason for this option.